irispy/iris_tools.py

```diff
@@ -139,14 +143,18 @@ def get_iris_response(time_obs=None, pre_launch=False, response_file=None, respo
         response_version_set = True
     else:
         response_version_set = False
-        if response_file_set is False:
-            response_version = 4
-    if response_file_set+pre_launch+response_version_set > 1:
+    if pre_launch_set + response_file_set + response_version_set == 2 or \
+        pre_launch_set + response_file_set + response_version_set == 3:
         raise ValueError("One and only one of kwargs pre_launch, response_file "
                          "and response_version must be set.")
     # If pre_launch set, define response_version to 2.
     if pre_launch:
         response_version = 2
+    if response_file_set is False:
+        try:
+            response_version = 4
+        except Exception:
+            raise("Version number not recognized.")
```

Comment:
In what case does this exception ever get raised?  Surely ```response_version = 4``` should always work?

Reply:
Yeah true. I should delete the exception part.